src/components/MenuContent/Menu.style.tsx

```diff
@@ -225,5 +225,4 @@ export const LogoWrapper = styled(Link)`
 
 export const NonWrappingSpan = styled.span`
   white-space: nowrap;
-  ${props => props.theme.fonts.regularBookMidWeight};
```

Comment:
ooo actually- separate of this pr, i think i have to fix the font weight of the `TextAndIconWithSpecialWrapping`